Geometry/MuonNumbering/src/GEMNumberingScheme.cc

```diff
@@ -35,7 +35,7 @@ int GEMNumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) const
 #ifdef EDM_ML_DEBUG
   if (levels != theRollLevel)
     edm::LogVerbatim("GEMNumberingScheme") << "MuonGEMNumberingScheme::BNToUN: BaseNumber has " << num.getLevels()
-                                           << " levels, need " << maxLevel << ":" << theRingLevel;
+                                           << " levels, need at least till " << theRingLevel;
```

Comment:
```suggestion
                                           << " levels, need at least till " << theRollLevel;
```

Reply:
It seems that the `if` check on line 36 should match the message.

Reply:
Changed the comment